translations/messages.fr.xlf

```diff
@@ -2287,6 +2295,82 @@
          Dans ce cas, le dernier joueur à avoir déplacé une pièce perd la partie.
          Ici, votre adversaire a déplacé sa pièce verte en dernier, vous êtes donc vainqueur !</target>
       </trans-unit>
+      <trans-unit id="2020802782007615776" datatype="html">
+        <source>Capture must be double diagonal steps</source>
```

Comment:
pas très user friendly cette phrase

Reply:
"just click on the green squares" :p ?

Reply:
"A capture should be a double diagonal step. Look at the green indicators to help you!" ?

Reply:
SIMPLE! BON! VOITURE!